menas/ui/components/schema/schemaMain.controller.js

```diff
@@ -104,14 +104,17 @@ sap.ui.controller("components.schema.schemaMain", {
 	},
 
 	resetNewSchemaValueState : function() {
-		sap.ui.getCore().byId("newSchemaName").setValueState(sap.ui.core.ValueState.None)
-		sap.ui.getCore().byId("newSchemaName").setValueStateText("")
+		sap.ui.getCore().byId("newSchemaName").setValueState(sap.ui.core.ValueState.None);
+		sap.ui.getCore().byId("newSchemaName").setValueStateText("");
+
+    this.byId("fileUploader").setValueState(sap.ui.core.ValueState.None);
+    this.byId("fileUploader").setValueStateText("");
 	},
 
 	validateNewSchema : function() {
 		this.resetNewSchemaValueState();
 		var schema = this._model.getProperty("/newSchema")
```

Comment:
If you are touching the function, please could you add the semicolon here too.

Reply:
I touched the function only to fix the semi-colon on the boolean variable. I also noticed that there is a lot places where the semi-colon is missing.

Reply:
fixed it for the validateNewSchema function.

Reply:
Yeah, I just wondered, why you fixed it for one line, and not the one above? :-)